.circleci/config.yml

```diff
@@ -6,12 +6,16 @@ jobs:
     steps:
       - checkout
       - run: pip install --user .
-      - run: sudo pip install coverage pylint==1.9.3
+      - run: sudo pip install coverage pylint==1.9.3 pycodestyle
       - run: make test
       - persist_to_workspace:
           root: .
           paths:
             - .coverage
+      - store_artifacts:
```

Comment:
They're "nice to have," as a reference point of the existing issues, if we choose to gradually improve conformance rather than a massive style auto-format.